bin/jsbq

```diff
@@ -57,14 +57,19 @@ jsbq.run = async () => {
     },
     d: {
       describe: 'BigQuery dataset name'
+    },
+    ignoreAdditional: {
+      describe: 'skip checking for additional properties in schemas'
     }
   })
   .argv
 
   const schemaData = await readFile(argv.j)
+  options = {};
```

Comment:
This and the next line can simply now be:
```
const options = {
  ignoreAdditional: argv.ignoreAdditional
}
```

Reply:
updated